src/jsx/ChatBox/test_message.jsx

```diff
@@ -0,0 +1,33 @@
+/** @jsx React.DOM */
```

Comment:
This was necessary for older versions of react or jsdom or node or something. So that it knows to compile it differently (won't error out on syntax error). I think now that we have a built in compiler, we might not really need this.